src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const peopleOfList = document.querySelectorAll('li');
+
+const listString = function(string) {
+  return +string.slice(1).split(',').join('');
+};
```

Comment:
will be better to use such syntax for object methods
if you want to create function and save it to the variable -> use arrow function
for other cases you can use function declaration and specify name after the `function` keyword